packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -578,7 +578,7 @@ export default function ImportTransactions({ modalProps, options }) {
   // options which are simple post-processing. That means if you
   // parsed different files without closing the modal, it wouldn't
   // re-read this.
-  let [csvDelimiter, setCsvDelimiter] = useState(
+  let [delimiter, setdelimiter] = useState(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏last change request, I promise :)

Should use camelCase here.
```suggestion
  let [delimiter, setDelimiter] = useState(
```